modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -5,10 +5,35 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Arr } from '@ephox/katamari';
 import Editor from 'tinymce/core/api/Editor';
-import { getToolbar } from '../api/Settings';
+import { Menu } from 'tinymce/core/api/ui/Ui';
+import { ClassList, getCellClassList, getTableClassList, getToolbar } from '../api/Settings';
 import { Clipboard } from '../core/Clipboard';
 import { SelectionTargets, LockedDisable } from '../selection/SelectionTargets';
+import { onSetupToggle } from './ButtonToggleUtils';
+
+const filterNoneItem = (list: ClassList) =>
+  Arr.filter(list, (item) =>
+    item.value !== ''
+  );
```

Comment:
```suggestion
  Arr.filter(list, (item) => item.value !== '');
```
Also, this should really be using `Strings.isNotEmpty(item.value)` instead, however I can't easily make that suggestion.

- [ ] Formatting and use `Strings.isNotEmpty` please